examples/guides/task_specific.md

```diff
@@ -20,7 +20,7 @@ The file should be named `AmazonPhotos.zip`, and it should be placed in `${task_
 
 The Bucc2018 and Tatoeba tasks are sentence retrieval tasks, and require the `faiss` library to run. `faiss-gpu` is recommended for speed reasons.
 
-We recommend running:
+We recommend running (with the appropriate `cudatoolkit` version):
```

Comment:
What dictates the "appropriate" version?

Reply:
Will do.

Reply:
I don't have anywhere to document this yet, but on prince/cassio you may need to module-load the appropriate CUDA libraries. `faiss` has caused me many headaches over time.